Dockerfile

```diff
@@ -64,6 +66,22 @@ RUN if [ "${PYTORCH}" = "true" ]; then\
       pip3 install torch==1.10.0+cu113 torchvision==0.11.1+cu113 torchaudio==0.10.0+cu113 -f https://download.pytorch.org/whl/cu113/torch_stable.html;\
     fi
 
+WORKDIR /opt/sofa
+RUN if [ "${SOFA}" == "false" ]; then\
+      echo 'Not installing Sofa';\
+    else\
+      echo 'Installing Sofa';\
+      wget https://github.com/sofa-framework/sofa/releases/download/v21.06.02/SOFA_v21.06.02_Linux.zip &&\
```

Comment:
@marcomerton Could you add the SOFA version as a build argument with a default value set to `SOFA_v21.06.02`? Then, it should be much easier to upgrade as the version is used in quite a few code lines below

Reply:
Done.
I also added the building instruction with (default) Sofa and with a specific Sofa version